scripts/set_gradle_proxy.sh

```diff
@@ -52,6 +52,19 @@ writeProxyStringToGradleProps () {
 	EOF
 }
 
+writeNoProxyEnvToGradleProps() {
+    local HOSTS=$1
+    local FILE=$2
+
+    grep -qF "systemProp.http.nonProxyHosts" $FILE 2>/dev/null && return 1
+
+    # Gradle / JVM expects a list separated by pipes instead of the comma that
+    # is used in shell environments
+    PREPED_HOSTS="$(echo ${HOSTS} | sed -e 's/,/|/g')"
+
+    echo "systemProp.http.nonProxyHosts=${PREPED_HOSTS}" >> $FILE
```

Comment:
We're already making use of Bash's string manipulation capabilities, so we could simply use `${HOSTS//,/|}` here.

But before that, could you do me a favor and add a separate commit before this one that changes the shebang lines in `import_proxy_certs.sh` and `set_gradle_proxy.sh` from `#!/bin/sh` to `#!/bin/bash`?

Reply:
> We're already making use of Bash's string manipulation capabilities, so we could simply use ${HOSTS//,/|} here.

Last time I tried I got a "bad substitution" error. But maybe that was related to the shebang.

Reply:
Done and done.

Reply:
Done